dandi/utils.py

```diff
@@ -616,23 +614,18 @@ def get_instance(dandi_instance_id):
         )
 
 
-def try_multiple(ntrials, retry, base):
```

Comment:
we didn't use this in any downstream code -- may be better should go through a round of deprecation?

Reply:
@yarikoptic None of the other repositories that I have cloned use this function.  I don't see the point in deprecating it instead of just deleting it immediately, and I think we should consider `utils.py` to be entirely an implementation detail that can be modified at will.